README.md

```diff
@@ -51,26 +55,28 @@ You're good to go! You can start learning the language by browsing the code and
 
 Yes, it's that easy. Whenever you make changes, it is a good idea to run `make clean` before `make rescue`.
 
-### To solve problems:
+The beginnings of a rock documentation (updated almost daily) is available at the [rock-dev repo](https://github.com/shamanas/rock-dev). It includes a lot more detail on how to build rock, as well as information on its code structure and where different parts are located. 
+
```

Comment:
I dunno, maybe it will be. But we should also have in mind that it can stop and stay as it is for weeks.